bioblend/galaxy/workflows/__init__.py

```diff
@@ -16,15 +16,25 @@ def get_workflows(self, workflow_id=None, name=None, deleted=False, published=Fa
         or ``workflow_id``. Provide only one argument, ``name`` or ``workflow_id``,
         but not both.
 
-        If ``name`` is set and multiple names match the given name, all the
-        workflows matching the argument will be returned.
+        :type workflow_id: str
+        :param workflow_id: Encoded workflow ID (incompatible with ``name``)
 
-        Return a list of JSON formatted dicts each containing basic information
-        about a workflow.
+        :type name: str
+        :param name: Filter by name of workflow (incompatible with
+          ``workflow_id``). If multiple names match the given name, all the
+          workflows matching the argument will be returned.
+
+        :type deleted: bool
+        :param deleted: If set to ``True``, return workflows that have been
+          deleted.
+
+        :type published: bool
+        :param published: If set to ``True``, return published workflows.
```

Comment:
There has been a merge problem here, please remove the documentation for ```deleted``` and ```published``` above.